Automatic Validation ended with:
> 2024-01-25T03:13:29.0893102Z ##[error] Manifest Error: DisplayVersion declared in the manifest has overlap with existing DisplayVersion range in the index. Existing DisplayVersion range in index:  [ [2023.3, 2023.3]]

(Automated response - build 655.)

--- a/manifests/g/Google/AndroidStudio/Canary/2023.3.1.6/Google.AndroidStudio.Canary.installer.yaml
+++ b/manifests/g/Google/AndroidStudio/Canary/2023.3.1.6/Google.AndroidStudio.Canary.installer.yaml
@@ -15,7 +15,6 @@ UpgradeBehavior: install
 ReleaseDate: 2024-01-24
 AppsAndFeaturesEntries:
 - DisplayName: Android Studio
-  DisplayVersion: "2023.3"
   ProductCode: Android Studio
 Installers:
 - Architecture: x64

